scss/vendors/_normalise.scss

```diff
@@ -0,0 +1,350 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
+
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
+  */
+
+html {
+	  line-height: 1.15; /* 1 */
+	    -webkit-text-size-adjust: 100%; /* 2 */
+    }
+
+    /* Sections
+       ========================================================================== */
+
+/**
+ * Remove the margin in all browsers.
+  */
+
+body {
+	  margin: 0;
+  }
+
+  /**
+   * Render the `main` element consistently in IE.
+    */
+
+main {
+	  display: block;
+  }
+
+  /**
+   * Correct the font size and margin on `h1` elements within `section` and
+    * `article` contexts in Chrome, Firefox, and Safari.
+     */
+
+h1 {
+	  font-size: 2em;
+	    margin: 0.67em 0;
+    }
+
+    /* Grouping content
+       ========================================================================== */
+
+/**
+ * 1. Add the correct box sizing in Firefox.
+  * 2. Show the overflow in Edge and IE.
+   */
+
+hr {
+	  box-sizing: content-box; /* 1 */
```

Comment:
Indentation in here looks a little wonky in places. Might be best downloading a copy as opposed to copying and pasting it.

Reply:
You right. I've `curl`'d it now so any further issues are problems with the actual stylesheet!!!